.circleci/config.yml

```diff
@@ -0,0 +1,140 @@
+            composer global require hirak/prestissimo
+            echo 'export PATH=~/.composer/vendor/bin:$PATH' >> $BASH_ENV
+      - run:
+          name: Edit MySQL Configuration
+          command: |
+            mysql -uroot -e "grant FILE on *.* to 'drupal'@'%'"
+            sudo chmod 777 /etc/mysql/my.cnf
+            sudo echo -e "[client]\nloose-local-infile" >> /etc/mysql/my.cnf
+            sudo chmod 644 /etc/mysql/my.cnf
+      - run:
+          name: Adjust PHP Settings
+          command: |
+            sudo chmod 777 /usr/local/etc/php/conf.d
+            echo "memory_limit = 256M" > /usr/local/etc/php/conf.d/memory.ini
+            echo "always_populate_raw_post_data = -1" > /usr/local/etc/php/conf.d/deprecated.ini
+
+      - checkout
+      - run: php -i
+      - run:
+          name: Setup Apache
+          command : |
+            sudo cp .circleci/tests/circle/circle.conf /etc/apache2/sites-available
+            sudo a2dissite 000-default
+            sudo a2ensite circle.conf
+            sudo a2enmod rewrite
+            sudo service apache2 restart
+
+      # Ahoy install in dkan-init.sh had problems detecting root user.
+      - run:
+          name: Install Ahoy
+          command: |
+            sudo wget -q https://github.com/devinci-code/ahoy/releases/download/1.1.0/ahoy-`uname -s`-amd64 -O /usr/local/bin/ahoy &&
+            sudo chmod +x /usr/local/bin/ahoy
+
+      ### Setup DKAN
+      - run:
+          name: DKAN Initialization
+          command: |
+            bash dkan-init.sh dkan --deps --build=$DATABASE_URL
+            ahoy drush --yes en dkan_harvest dkan_harvest_datajson dkan_harvest_dashboard
+            ahoy drush cc all
+      - run:
+          name: Server Router Background Server Task
+          command: ahoy dkan server
+          background: true
+      - run:
+          name: X11 VNC Background Server Task
+          command: x11vnc -forever -nopw
+          background: true
+
+      - run:
+          name: Enable dkan_harvest_test and reinstall
+          command: |
+            ahoy drush en --yes dkan_harvest_test
+            ahoy dkan reinstall --yes
+      - save_cache:
+          key: v1-dkan-{{ .Branch }}
+          paths:
+            - ~/.drush
+            - ~/.composer
+      - save_cache:
+          key: v1-dkan-test-vendor
+          paths:
+            - dkan/test/vendor
+      - run:
+          name: Run Parallel Behat Tests
+          command: ruby dkan/.ahoy/.scripts/circle-behat.rb docroot/profiles/dkan/test/features
+      # Lint only on 1st parallel instance. Deploy: blocks until parallelism is done, so doing it this way.
+      - run:
+          name: Run Lint
+          command: |
+            if [ "$CIRCLE_NODE_INDEX" -eq "0" ]; then
+              ahoy dkan lint
+            fi
+      # Unit Tests only on 1st parallel instance. Deploy: blocks until parallelism is done, so doing it this way.
+      - run:
+          name: Run Unit Tests
```

Comment:
Sorry to nitpick but can we call these PHPUnit Tests? They're not really "unit tests" - they're functional tests in PHPUnit.

Reply:
Done - ahoy dkan unittest should probably be changed, but not today :)